mistlib/core.py

```diff
@@ -303,6 +303,99 @@ def write_pdf(self, file):
 
         return
     
+    def write_adamantine_input(self, file):
+        reference_temperature = self.properties["solidus_eutectic_temperature"].value #this was done for 3dthesis writer prbably can be removed?
+
+        p = self.properties["specific_heat_solid"]
```

Comment:
Looks like some of these properties overlap with 3dthesis - you could create some new `get_property` functions to reduce the repeated code

Reply:
That's a good idea -- there should be a function to get the value of a property at a given temperature that hides how it's generated (from a constant value, from a polynomial, from interpolating from a table).

Let's keep that out of scope for this PR, but I'm creating an issue for it.

Reply:
That's now #17.